avaframe/com1DFA/com1DFA.py

```diff
@@ -1607,7 +1607,13 @@ def initializeResistance(cfg, dem, simTypeActual, resLine, reportAreaInfo, thres
     reportAreaInfo: dict
         simulation area information dictionary completed with entrainment area info
     """
-    cRes = cfg.getfloat("cRes")
+    # TODO: if we keep the ResCoulomb - resistance types,
+    # we should initialize the Coulomb-Value (as the resistamce parameter)
+    ResModel = cfg["ResistanceModel"]
+    if ResModel in ["cRes", "cResCoulomb"]:
```

Comment:
To avoid potential problems with typos, use .lower to compare (ie. if user writes cRescoulomb (lower c), it should work too)